docs/getting_started/features/accesibility.md

```diff
@@ -5,48 +5,171 @@ GTFS contains multiple features intended to assist riders in navigating and acce
 
 Wheelchair accessibility fields make it possible to indicate if a stop and vehicle can accommodate users using wheelchairs, allowing them to plan their trips based on the most convenient option for their needs.
 
+**Pre-requirements**: Implement Base features, if provding wheelchair accesibility for station entrances and exits Location Types features is also required.
+
 | Files included                   | Fields included   |
 |----------------------------------|-------------------|
-|[stops.txt](/schedule/reference/#stopstxt)|`wheelchair_boarding` |
-|[trips.txt](/schedule/reference/#tripstxt)|`wheelchair_accessible`|
+|[stops.txt](/documentation/schedule/reference/#stopstxt)|`wheelchair_boarding` |
+|[trips.txt](/documentation/schedule/reference/#tripstxt)|`wheelchair_accessible`|
+
+??? note "Sample Data"
+    <p style="font-size:16px">
+    The following tables indicate that in stop `TAS001` some vehicles can be boarded by a rider in a wheelchair `wheelchair_boarding=1` in `stops.txt` while using `trips.txt` it is specified that the vehicle providing service for trip `AWE1` can accommodate at least one rider in a wheelchair, while the vehicle used for trip `AWE2` cannot. 
```

Comment:
This is a bit of a long sentence, there might be a better way to explain that both the trip and the stop need to be wheelchair accessible. 

Here is an alternative suggestion:

```suggestion
    The following sample shows wheelchair boarding is available at stop `TAS001` with  `wheelchair_boarding=1`.
The vehicle used in trip `AWE1` is equipped to accommodate at least one wheelchair, and the vehicle used in trip `AWE2` is not.
Both the stop and trip must be wheelchair accessible for a passenger to be able to access a trip at the given stop.